### PR TITLE
fix missing pagination in search results

### PR DESCRIPTION
**Pull request by sanderha**
Results in SearchForm did not take start GET var into account

### CODE REVIEW

**Comment by sanderha** [comment]
#228

File: src/Solr/Forms/SearchForm.php [hunk 1]
```diff
@@ -63,7 +63,10 @@ public function getResults()
         $request = $this->getRequestHandler()->getRequest();
 
         $searchTerms = $request->requestVar('Search');
-        $query = SearchQuery::create()->addSearchTerm($searchTerms);
+        // for pagination
+        $start = $request->requestVar('start');
```
Comment (robbieaverill) [inline src/Solr/Forms/SearchForm.php]: Perhaps we should check whether it has a value before setting it, that way we still get the default value if no argument is provided

**Comment by sanderha** [comment]
@robbieaverill added a conditional to check for the start GET variable first

File: src/Solr/Forms/SearchForm.php [hunk 2]
```diff
@@ -65,6 +65,10 @@ public function getResults()
         $searchTerms = $request->requestVar('Search');
         $query = SearchQuery::create()->addSearchTerm($searchTerms);
 
+        if($start = $request->requestVar('start')){
```
Comment (robbieaverill) [inline src/Solr/Forms/SearchForm.php]: Just PSR-2 compliance to go, fixing now for you

**Comment by robbieaverill** [comment]
I've noticed that you've created this pull request from one of the "core branches" rather than a branch specifically intended for this pull request. This can cause problems if we ask you to target this PR at a different branch. No action is needed at the moment, this is for your information.

In the future please create a new branch for each individual pull request. See our [guide about contributing PRs](https://docs.silverstripe.org/en/4/contributing/code/#step-by-step-from-forking-to-sending-the-pull-request)

**Comment by robbieaverill** [comment]
Thank you!

**Comment by sanderha** [comment]
Ah yeah, sorry about that. Will think about it next time. The more you know :-)

**Pull request merged**